.snyk

```diff
@@ -0,0 +1,12 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.1
+ignore: {}
+patch: {}
+exclude:
+  global:
+    - src/test
+    - src/integrationTest
+    # [CWE-547] Hardcoded Secret - Legacy code from PUI
+    - src/main/java/uk/gov/laa/ccms/caab/opa/security/Encryptor.java
+    # [CWE-352] Cross-Site Request Forgery (CSRF) - To be implemented: Ticket TBC
```

Comment:
Want me to create a ticket for this

Reply:
Created https://dsdmoj.atlassian.net/browse/CCMSPUI-499